build.gradle

```diff
@@ -81,6 +80,13 @@ subprojects {
     }
   }
 
+  // String interpolation support, see https://github.com/antkorwin/better-strings
+  dependencies {
+    def betterStrings = [ group: 'com.antkorwin', name: 'better-strings', version: '0.2' ]
```

Comment:
Maybe also extract version to variable and then move this section to `ext` section if possible?

Reply:
Done